gapis/resolve/dependencygraph2/graph_visualization/graph_visualization.go

```diff
@@ -19,33 +19,62 @@ import (
 	"fmt"
 	"github.com/google/gapid/gapis/api"
 	"github.com/google/gapid/gapis/resolve/dependencygraph2"
+	"github.com/google/gapid/gapis/service"
 	"github.com/google/gapid/gapis/service/path"
 )
 
+const (
+	EMPTY = "Empty"
+)
+
 func createGraphFromDependencyGraph(dependencyGraph dependencygraph2.DependencyGraph) (*graph, error) {
 
 	currentGraph := createGraph(0)
-	currentHierarchy := &api.Hierarchy{}
+	idToBuilder := map[api.ID]api.GraphVisualizationBuilder{}
 
 	err := dependencyGraph.ForeachNode(
 		func(nodeId dependencygraph2.NodeID, dependencyNode dependencygraph2.Node) error {
 			if cmdNode, ok := dependencyNode.(dependencygraph2.CmdNode); ok {
 				cmdNodeId := cmdNode.Index[0]
 				command := dependencyGraph.GetCommand(api.CmdID(cmdNodeId))
-				commandName := command.CmdName()
+				isReal := api.CmdID(cmdNodeId).IsReal()
+				if !isReal {
```

Comment:
Nit: no need for the `isReal` temporary variable.

Reply:
okay, I fixed it.